scootapi/client/run_job_cmd.go

```diff
@@ -33,13 +33,15 @@ func (c *runJobCmd) registerFlags() *cobra.Command {
 
 // Types to handle JobDefinitions from JSON files
 type JobDef struct {
-	Tasks map[string]TaskDef
+	Tasks     map[string]TaskDef
+	TimeoutMs int32
```

Comment:
Agreed on defaultTaskTimeout - it's just a shorthand vs including the field in each task. Limiting job time would be trickier, and different use case.